packages/react-tabs/src/components/Pivot/Pivot.base.tsx

```diff
@@ -259,7 +262,9 @@ export const PivotBase: React.FunctionComponent<IPivotProps> = React.forwardRef<
           {items}
           {overflowBehavior === 'menu' && (
             <CommandButton
-              className={classNames.overflowMenuButton}
+              // TODO when the span wrapper is removed from <CommandButton>, set
+              // className={classNames.link + ' ' + classNames.overflowMenuButton}
+              className={classNames.link}
```

Comment:
It turns out that my original workaround didn't correctly apply hover visuals when when the links were displayed as tabs. This new workaround is a more straightforward fix, which applies different classes to the wrapper `<span>` and the inner `<button>`.

@czearing FYI with this change, you'll now need to add both classes (`link` and `overflowMenuButton`) to the button once  you remove the span.